cmd/network-console-collector/profiles.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"github.com/skupperproject/skupper/cmd/network-console-collector/internal/flowlog"
+	"github.com/skupperproject/skupper/pkg/vanflow"
+)
+
+var (
+	// loggingProfileMinimal logs 1 vanflow event per second (with bursts up to 32)
+	// reduces Link Record noise to 1 every ~20s.
```

Comment:
@ted-ross thought you may find this interesting. I know you've mentioned link record noise before.

I felt I had to suppress logging for LinkRecord messages in particular due to the constant churn of octets/octet rates. When under load, I think Connector and Listener records were also observed to be particularly busy.